### ISSUE TITLE
Texts are overlapped in long titles

### ISSUE THREAD

**Issue by mielamvn** [fossasia/open-event-server#4693]
**I'm submitting a ...**  (check one with "x")
- [ ] bug report

**Current behavior:**
 Texts are overlapped in long titles on the main page
![texts overlapped](https://user-images.githubusercontent.com/36442723/39615444-66e74f06-4fa0-11e8-9e4f-f0eb1682ecf6.png)

Your operating system
```Linux Mint

**Comment by bhaveshAn** [comment]
I think it is issue of legacy repo and frontend repo (if it persists in open-event-frontend as well)

**Comment by mayank8318** [comment]
@bhaveshAn Right, I think this definitely is a front-end issue. @mielamvn Can you please confirm as to which(dev or master) deployments did you use? I tried to replicate the issue on my local server and found out that now instead of overlapping, the extra title is just cut and not shown at all :
![screenshot from 2018-05-04 16-13-52](https://user-images.githubusercontent.com/22395998/39624388-bcdc6a02-4fb6-11e8-9d7e-283d0e904eb7.png)

@bhaveshAn Can you confirm this?

**Comment by poush** [comment]
This issue is related with legacy repo.

@open-event-bot move to @fossasia/open-event-legacy
